src/current/v23.1/role-based-audit-logging.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Role-based SQL Audit Logging
+summary: Use the sql.log.user_audit cluster setting to turn SQL audit logging on or off for a user or role.
+toc: true
+docs_area: manage
+---
+
+Role-based SQL audit logging gives you detailed information about queries being executed against your system by specific users or roles. An event of type [`role_based_audit_event`]({% link {{ page.version.version }}/eventlog.md %}#role_based_audit_event) is recorded when an executed query belongs to a user whose role membership corresponds to a role that is enabled to emit an audit log via the [`sql.log.user_audit` cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-log-user-audit). The event is logged in the [`SENSITIVE_ACCESS`]({% link {{ page.version.version }}/logging.md %}#sensitive_access) logging channel.
+
+{% include enterprise-feature.md %}
+
+This page shows how to enable role-based SQL audit logging in CockroachDB, including:
+
+- How to configure audit logging.
+- Where the audit log files live.
+- What the audit log files look like.
+
+{{site.data.alerts.callout_info}}
+Logging has performance implications. This feature enables you to log only what you require allowing you to limit logging and reduce impact to your workload.
```

Comment:
```suggestion
Logging can negatively impact performance. Log only what you require to limit impact to your workload.
```

Reply:
@kevin-v-ngo you previously asked me not to use the phrase "negatively impact performance", in favor of "has performance implications". Is this still your preference?

Reply:
reviewed with @kevin-v-ngo. He clarified that he meant:

```
Logging, in general, can negatively impact performance. Log only what you require to limit impact to your workload.
```

updated with above line.